stable/database/files/nuobackup

```diff
@@ -184,7 +204,7 @@ fi
 if [ "$backup_type" != "full" ]; then
    error="$( nuodocker get current-backup \
       --db-name "${db_name}" \
-      --labels "backup ${backup_group} ${labels}" 2>&1)"
+      --labels "${labels}" 2>&1)"
```

Comment:
I think this is correct not to restrict to the backup group, but why was it there before?

Reply:
The backup group (if exist) is already included in the labels on line 164 in the script. It was incorrectly included again here. We should use here the same labels that we are using to issue the hotcopy because they define the `archiveIds` that will be selected for the hotcopy operation.